docs/apis-tools/zeebe-api/gateway-service.md

```diff
@@ -160,8 +160,6 @@ message CancelProcessInstanceRequest {
   // the process instance key (as, for example, obtained from
   // CreateProcessInstanceResponse)
   int64 processInstanceKey = 1;
-  // a reference key chosen by the user and will be part of all records resulting from this operation
-  optional uint64 operationReference = 2;
```

Comment:
I noticed these reference keys. Is there more context here? Is this not something we want to recommend or is there an alternative now?

Reply:
These were removed in the protobuf, but I'll leave it up to @korthout if we want to keep this in the docs?

Reply:
CC @megglos

Reply:
@akeller As @megglos is out on parental leave for about a month, I think we can go ahead and merge this. If he returns and decides to add the reference keys we can certainly pop them back in!